opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -1178,7 +1178,7 @@ namespace Opm {
     computeWellPotentials(std::vector<double>& well_potentials, const int reportStepIdx, Opm::DeferredLogger& deferred_logger)
     {
         // number of wells and phases
-        const int nw = numLocalWells();
```

Comment:
here, commenting out the well potential calculation for now. Because it stops the running of the test model. 

This also makes the PR not mergable before we address the well potential calculation.

Reply:
Is it potentials for MSW that stop it? In that case hopefully the work that is on its way in can help. Injector or producer?

Reply:
producer. I  am still testing and will provide more information.

Reply:
The problem is fixed.